pkg/common/utils/utils.go

```diff
@@ -223,3 +226,61 @@ func QueryVolumeDetailsUtil(ctx context.Context, m cnsvolume.Manager, volumeIds
 	}
 	return volumeDetailsMap, nil
 }
+
+// LogoutAllSessions will loggout all vCenter sessions and disconnect vCenter client
+func LogoutAllSessions(ctx context.Context, isMultiVCenterFssEnabled bool) error {
+	log := logger.GetLogger(ctx)
+	log.Info("logging out vCenter session")
+	virtualcentermanager := cnsvsphere.GetVirtualCenterManager(ctx)
+	vCenters := virtualcentermanager.GetAllVirtualCenters()
+	for _, vc := range vCenters {
+		if vc.Client == nil {
+			continue
+		}
+		log.Info("closing idle vCenter session")
+		vc.Client.CloseIdleConnections()
+		// logout vCenter session for list-view
+		mgr, err := cnsvolume.GetManager(ctx, vc, nil, false, isMultiVCenterFssEnabled, false, false)
+		if err != nil {
+			log.Errorf("failed to create an instance of volume manager, err=%v", err)
+			return err
+		}
+		err = mgr.LogoutListViewVCSession(ctx)
+		if err != nil {
+			log.Errorf("Error while logout vCenter session for list-view, Error: %+v", err)
+			return err
```

Comment:
when error happens continue with the next vCenter and try to disconect sessions for rest of the vcenters